operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -201,9 +201,18 @@ limits_config:
 {{- end }}
 {{- end }}{{- end }}
   max_cache_freshness_per_query: 10m
-  per_stream_rate_limit: {{ .Stack.Limits.Global.IngestionLimits.PerStreamRateLimit }}MB
-  per_stream_rate_limit_burst: {{ .Stack.Limits.Global.IngestionLimits.PerStreamRateLimitBurst }}MB
+{{- with .Stack.Limits.Global.IngestionLimits.PerStreamRateLimit }}
+  per_stream_rate_limit: {{ . }}MB
+{{- end }}
+{{- with .Stack.Limits.Global.IngestionLimits.PerStreamRateLimitBurst }}
+  per_stream_rate_limit_burst: {{ . }}MB
+{{- end }}
```

Comment:
Doesn't the other conversation mean that these limits are not optional at all and the operator should expect a limit and burst limit to be configured (even if it is just the default)?

Reply:
Actually the `sizes.go` declares a new default per-stream-rate limit of 5MB and leaves the default for burst to the Loki config. WDYT? (maybe just removing the template conditional for `per_stream_rate_limit` is enough?!?)